source/includes/_ios_delegate.md

```diff
@@ -0,0 +1,111 @@
+# Survey Delegate
+
+The `WTRSurveyDelegate` can be used to be informed of when the SDK will or
+already perform certain actions. To use the delegate, first a class needs
+to conform to the protocol and assign that class to the WootricSDK.
+
+Imagine you have a view controller `SampleViewController.m`. You can
+specific that this class conforms to `WTRSurveyDelegate` and it will be
```

Comment:
Maybe there's a better way to say this, like maybe use the word inheritance or inherit.